### ISSUE TITLE
Is it possible to obtain CacheSettings from Deno itself

### ISSUE THREAD

**Issue by jollytoad** [denoland/deno_cache#19]
It would be very useful to allow `CacheSettings` to default to whatever Deno itself is doing, for example, if `--reload` has been passed to reload all, or with a list of modules.

Is this exposed by Deno?

**Comment by dsherret** [comment]
No, it's not exposed. It's probably best to add a `--reload` flag to the tool using this. For example, the user might not want to reload the tool's code, but they do want to reload the dependencies being cached with the tool.
